packages/mask/src/plugins/Trader/SNSAdaptor/trader/Trader.tsx

```diff
@@ -320,7 +318,7 @@ export function Trader(props: TraderProps) {
     // #endregion
 
     // Query the balance of native tokens on target chain
-    useUpdateBalance(chainId, currentChainId)
+    useUpdateBalance(chainId)
```

Comment:
Why remove `currentChainId`

Reply:
`chainId` is result of `targetChainId ?? currentChainId`, no need to pass `currentChainId` to `useUpdateBalance`, also reduce concepts in`useUpdateBalance`